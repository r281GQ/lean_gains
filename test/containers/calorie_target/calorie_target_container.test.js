import React from 'react';
import { shallow } from 'enzyme';

import CalorieTargetContainer from './../../../src/containers/calorie_target/calorie_target_container';

describe('CalorieTargetContainer test', () => {
  it('should render', () => {
    const component = shallow(
      <CalorieTargetContainer/>
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
