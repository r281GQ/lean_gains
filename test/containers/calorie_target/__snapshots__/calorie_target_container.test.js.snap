// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalorieTargetContainer test should render 1`] = `
<div
  className="calorie-target"
>
  <div
    className="row"
  >
    <div
      className="col col-6"
    >
      <Connect(ReduxForm) />
    </div>
    <div
      className="col col-6"
    >
      <Connect(CalorieTargetResultContainer) />
    </div>
  </div>
  ;
</div>
`;
